cmd/benchmark.go

```diff
@@ -385,6 +399,129 @@ func getPackageNameAndVersion(packageFromRegistry string) (string, string, error
 	return packageName, packageVersion, nil
 }
 
+func getStreamCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "stream",
+		Short: "Run stream benchmarks",
+		Long:  "Run stream benchmarks for the package",
+		Args:  cobra.NoArgs,
+		RunE:  streamCommandAction,
+	}
+
+	cmd.Flags().StringP(cobraext.BenchNameFlagName, "", "", cobraext.BenchNameFlagDescription)
+	cmd.Flags().String(cobraext.VariantFlagName, "", cobraext.VariantFlagDescription)
+	cmd.Flags().DurationP(cobraext.BenchStreamBackFillFlagName, "", 15*time.Minute, cobraext.BenchStreamBackFillFlagDescription)
+	cmd.Flags().Uint64P(cobraext.BenchStreamEventsPerPeriodFlagName, "", 10, cobraext.BenchStreamEventsPerPeriodFlagDescription)
+	cmd.Flags().DurationP(cobraext.BenchStreamPeriodDurationFlagName, "", 10*time.Second, cobraext.BenchStreamPeriodDurationFlagDescription)
+	cmd.Flags().BoolP(cobraext.BenchStreamPerformCleanupFlagName, "", false, cobraext.BenchStreamPerformCleanupFlagDescription)
+	cmd.Flags().StringP(cobraext.BenchStreamTimestampFieldFlagName, "", "timestamp", cobraext.BenchStreamTimestampFieldFlagDescription)
+
+	return cmd
+}
+
+func streamCommandAction(cmd *cobra.Command, args []string) error {
+	cmd.Println("Run stream benchmarks for the package")
+
+	variant, err := cmd.Flags().GetString(cobraext.VariantFlagName)
+	if err != nil {
+		return cobraext.FlagParsingError(err, cobraext.VariantFlagName)
+	}
+
+	benchName, err := cmd.Flags().GetString(cobraext.BenchNameFlagName)
+	if err != nil {
+		return cobraext.FlagParsingError(err, cobraext.VariantFlagName)
```

Comment:
Oh, my suggestion was incorrect, sorry :facepalm:
```suggestion
		return cobraext.FlagParsingError(err, cobraext.BenchNameFlagName)
```